core/players-handbook-2024/races/race-human.xml

```diff
@@ -0,0 +1,78 @@
+			
+			<h4>Human Traits</h4>
+			<p style="margin-top:15px"><strong><em>Creature Type:</em></strong> Humanoid</p>
+			<p><strong><em>Size:</em></strong> Medium (about 4–7 feet tall) or Small (about 2–4 feet tall), chosen when you select this species</p>
+			<p><strong><em>Speed:</em></strong> 30 feet</p>
+			
+			<p>As a Human, you have these special traits.</p>
+			<p class="indent"><strong><em>Resourceful.</em></strong> You gain Heroic Inspiration whenever you finish a Long Rest.</p>
+			<p class="indent"><strong><em>Skillful.</em></strong> You gain proficiency in one skill of your choice.</p> 
+			<p class="indent"><strong><em>Versatile.</em></strong> You gain an Origin feat of your choice. Skilled is recommended.</p> 
+		</description>
+		<sheet display="false" />
+		<rules>
+			<grant type="Grants" id="ID_INTERNAL_GRANT_RACE_HUMAN" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement Option (Human)" supports="Custom Ability Score Improvement Option" requirements="!ID_INTERNAL_GRANTS_BACKGROUND_ASI" />
+			<select type="Racial Trait" name="Size (Human)" supports="ID_INTERNAL_RACIAL_TRAIT_SMALL|ID_INTERNAL_RACIAL_TRAIT_MEDIUM" />
+			<stat name="innate speed" value="30" bonus="base" />
+
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language (Human)" supports="Custom Race Language" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Language (Human)" supports="Standard||Exotic" number="2" />
+
+			<grant type="Racial Trait" id="ID_WOTC_PHB24_RACIAL_TRAIT_HUMAN_RESOURCEFUL" requirements="!ID_WOTC_PHB24_RACIAL_TRAIT_REPLACEMENT_HUMAN_RESOURCEFUL" />
+			<grant type="Racial Trait" id="ID_WOTC_PHB24_RACIAL_TRAIT_HUMAN_SKILLFUL" requirements="!ID_WOTC_PHB24_RACIAL_TRAIT_REPLACEMENT_HUMAN_SKILLFUL" />
+			<grant type="Racial Trait" id="ID_WOTC_PHB24_RACIAL_TRAIT_HUMAN_VERSATILE" requirements="!ID_WOTC_PHB24_RACIAL_TRAIT_REPLACEMENT_HUMAN_VERSATILE" />
+		</rules>
+	</element>
+
+	<element name="Resourceful" type="Racial Trait" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_RACIAL_TRAIT_HUMAN_RESOURCEFUL">
+		<description>
+			<p>You gain Heroic Inspiration whenever you finish a Long Rest.</p> 
+		</description>
+		<sheet>
+			<description>You gain Heroic Inspiration whenever you finish a Long Rest.</description>
+		</sheet>
+	</element>
+
+	<element name="Skillful" type="Racial Trait" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_RACIAL_TRAIT_HUMAN_SKILLFUL">
+		<description>
+			<p>You gain proficiency in one skill of your choice.</p> 
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Proficiency" name="Skill Proficiency (Skillful)" supports="Skill" />
```

Comment:
With just `Skill` as the support, it allows the PC to choose an expertise option. Need to add `!Expertise` so we don't have those show up. 
```suggestion
			<select type="Proficiency" name="Skill Proficiency (Skillful)" supports="Skill,!Expertise" />
```